packages/gatsby-theme-docs/src/hooks/use-resized-images.js

```diff
@@ -0,0 +1,44 @@
+import nodeJsPath from 'path';
+import { useStaticQuery, graphql } from 'gatsby';
+
+export const useResizedImages = () => {
+  const queryResult = useStaticQuery(
+    graphql`
+      {
+        allImageSharp {
+          nodes {
+            parent {
+              ... on File {
+                relativePath
+                relativeDirectory
+              }
+            }
+            fluid(maxWidth: 754, srcSetBreakpoints: [754, 1508]) {
+              srcSet
+              src
+              presentationWidth
+              presentationHeight
+              originalImg
+            }
+          }
+        }
+      }
+    `
+  );
+  return queryResult.allImageSharp.nodes.map((node) => {
+    return {
+      ...node.fluid,
+      path: nodeJsPath.join(
+        '/images/',
+        node.parent.relativeDirectory,
+        node.parent.relativePath
+      ),
+    };
+  });
+};
+
+export const useResizedImagesByPath = (path = '') => {
```

Comment:
Why the empty string as default value?

```suggestion
export const useResizedImagesByPath = (path) => {
```